src/AzureStorageService.js

```diff
@@ -23,20 +27,30 @@ export class AzureStorageService extends BaseStorageService {
       throw new Error('Azure__StorageService :: Required configuration is missing');
     }
     this.reportsContainer = _.get(config, 'reportsContainer')?.toString();
-    this.blobService = azure.createBlobService(config?.identity, config?.credential);
+    this.sharedKeyCredential = new StorageSharedKeyCredential(config?.identity, config?.credential);
+    this.blobService = new BlobServiceClient(
```

Comment:
@abhishekpnt please add in try catch in case of exception
CC: @rajeevsathish

Reply:
Added Azure client creation in try-catch block